docs/04-resources/06-guides/rsk-cli/deploy.md

```diff
@@ -0,0 +1,223 @@
+    }
+
+    // Function to set the name and phone
+    function setContactInfo(string memory _name, string memory _phone) public {
+        name = _name;
+        phone = _phone;
+    }
+
+    // Function to get the contact info
+    function getContactInfo() public view returns (string memory, string memory) {
+        return (name, phone);
+    }
+}
+```
+
+2. **Compile the Contract**:
+
+Run the following command to compile your smart contract:
+
+```
+npx hardhat compile
+```
+
+You should see a response similar to this:
+
+```
+Generating typings for: 2 artifacts in dir: typechain-types for target: ethers-v6
+Successfully generated 8 typings!
+Compiled 2 Solidity files successfully (evm target: paris).
+```
+
+This command compiles the project, generating an artifacts folder that contains the compiled data, including the ABI and bytecode for your contract.
+
+```
+├── artifacts
+│   ├── build-info
+│   │   └── f5da7ce57199502ee2303fea40...
+│   └── contracts
+│  │   └── ContactInfo.sol
+│            ├── ContactInfo.dbg.json
+│            └── ContactInfo.json
+│   
+├── cache
+├── contracts
+│   ├── ContactInfo.sol
+│   └── Lock.sol
+├── ignition
+├── node_modules
+├── test
+├── typechain-types
+├── .gitignore
+├── hardhat.config.ts
+├── package-lock.json
+├── package.json
+├── README.md
+└── tsconfig.json
+```
+
+###
+ **Explore the Artifacts Folder**:  
+  * Inside the artifacts folder, you’ll find two subfolders:  
+    * `build-info`  
+    * `contracts`  
+  > **Note**: Ensure that the contract’s file name matches the contract’s name in the Solidity code.  
+- **Locate the ABI and Bytecode**:  
+ 
+    :::info[Info]
+
+     * **ABI**: The Application Binary Interface (ABI) defines the functions, events, and types in the contract, enabling interaction with it.  
+     * **Bytecode**: The bytecode is the compiled code that will run on the Ethereum Virtual Machine (EVM). This code is deployed to the blockchain.  
+
+    :::
+
+  * Inside the contracts folder, open your contract’s folder (**ContactInfo** in this case). You will see two files:  
+    * `ContactInfo.dbg.json`  
+    * `ContactInfo.json`  
+   
+ 
+- **Copy the ABI and Bytecode**:  
+  * Open the ContactInfo.json file. Locate and copy the abi and bytecode, which will be used for deployment.
+
+<figure>
+<img src="/img/guides/rsk-cli/artifacts-folder.png" alt="Hardhat Artifacts Folder"/>
+  <figcaption>Hardhat Artifacts Folder (fig 1.)</figcaption>
+</figure>
+
+ Save the ABI and bytecode in a new folder. You can name the folder anything, but for this tutorial, you will call it file. Inside this folder:
+
+> * Save the ABI as `abi.json`.  
+> * Save the bytecode as `bytecode.bin` (make sure to copy the bytecode without quotes).
+
+You will use `.json` for the ABI file because the ABI is structured in JSON format, detailing the functions, events, and types used in the contract. This format is easy for applications to parse and use for interacting with the contract.
```

Comment:
```suggestion
Use `.json` for the ABI file because the ABI is structured in JSON format, detailing the functions, events, and types used in the contract. This format is easy for applications to parse and use for interacting with the contract.
```